charts/rancher-compliance/107.0.0+up1.0.0-rc.1/Chart.yaml

```diff
@@ -0,0 +1,22 @@
+annotations:
+  catalog.cattle.io/auto-install: rancher-compliance-crd=match
+  catalog.cattle.io/certified: rancher
+  catalog.cattle.io/display-name: Rancher Compliance
+  catalog.cattle.io/kube-version: '>= 1.31.0-0 < 1.34.0-0'
+  catalog.cattle.io/namespace: rancher-compliance-system
+  catalog.cattle.io/os: linux
+  catalog.cattle.io/permits-os: linux,windows
+  catalog.cattle.io/provides-gvr: compliance.cattle.io.clusterscans/v1
+  catalog.cattle.io/rancher-version: '>= 2.12.0-0 < 2.13.0-0'
+  catalog.cattle.io/release-name: rancher-compliance
+  catalog.cattle.io/type: cluster-tool
+  catalog.cattle.io/ui-component: rancher-compliance
+apiVersion: v1
+appVersion: v1.0.0-rc.1
+description: The compliance-operator enables running security scans on a kubernetes
+  cluster
+icon: file://assets/logos/rancher-compliance.svg
```

Comment:
Don't we need to also update the [file name](https://github.com/rancher/compliance-operator/blob/406ee580a2cbc9f16d2a946f5cd9eb14529babca/chart/Chart.yaml#L18) in the upstream chart? At the moment it is referring to a `compliance.svg` as opposed to `rancher-compliance.svg`.